### PR TITLE
Use InClusterConfig() if kubeconfig is not found

### PR DESCRIPTION
**Pull request by bugfest**
Use InClusterConfig() if kubeconfig is not found in home.
Added kubernetes pod example

### CODE REVIEW

**Comment by bugfest** [comment]
Ok... so, would you so kind to merge this one? :_D

**Comment by bjhaid** [comment]
@bugfest I would merge it now I somehow never noticed the PR

**Comment by bjhaid** [comment]
thanks! I'll publish a new docker image as soon as possible

File: examples/k8s-example.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+kind: ClusterRole
+metadata:
+  name: node-labels-to-files
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
```
Comment (bjhaid) [inline examples/k8s-example.yaml:19]: needs `list` and `watch` for the `always` mode, if you are feeling like it you can open a PR or I'll do that if I do not see a PR by EoD my time.

**Comment by bjhaid** [comment]
FYI I published a new version with your work today, thanks for your contribution!

**Pull request merged**